llvm/test/Transforms/LoopVectorize/simple_early_exit.ll

```diff
@@ -918,6 +917,189 @@ loop.end:
 }
 
 
+define i64 @multiple_exiting_one_early_same_exit_phi_of_consts() {
```

Comment:
This file is getting a bit big , perhaps worth splitting it up into multiple files if possible?

Reply:
Sure, that makes sense. If it's ok with you, I'll land this patch first then follow up with a simple refactor that collects tests of the same class in their own file, i.e. early exits with no outside users of loop variables, ones that use loop variables, tests with multiple early exits (countable + uncountable), tests for different memory access patterns, etc.